docs/volto_customization/installation.md

```diff
@@ -17,150 +17,124 @@ Getting started with Volto involves setting up a development environment, unders
 
 ## Prerequisites
 
-Before you start working with Volto, ensure you have the following prerequisites:
+Before you start working with this training, ensure you have the following prerequisites:
 
-- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (16.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
+- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (>=20.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
 - <a target="_blank" href="https://www.python.org/">Python</a> - See below for specific versions.
+- <a target="_blank" href="https://pipx.pypa.io/stable/">pipx</a>
+- <a target="_blank" href="https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating">nvm</a>
+- <a target="_blank" href="https://www.gnu.org/software/make/">GNU make</a>
 - <a target="_blank" href="https://www.docker.com/get-started">Docker</a> (if using the Plone docker images - <a target="_blank" href="https://6.docs.plone.org/install/containers/index.html">see instructions for installation and usage</a>)
 
```

Comment:
I would suggest that you replace all the above with a link to https://6.docs.plone.org/install/create-project-cookieplone.html#prerequisites-for-installation. This uses Intersphinx and MyST syntax. Then you'll never have to update it again. 😁 

Also it is not good for accessibility to force users to open links in a new browser window, especially for web app developers who know what they are doing while browsing. I personally despise that behavior, and prefer to command-click, right-click and open in new window, or use my mouse's scroll wheel click.

```suggestion
Follow the guide in Plone 6 Documentation, {ref}`create-project-cookieplone-prerequisites-for-installation-label`.
```